CVEs/CVE-2021-31712.json

```
@@ -0,0 +1,30 @@
+{
+    "CVE": "CVE-2021-31712",
+    "state": "PUBLISHED",
+    "repository": ["https://github.com/jpuri/react-draft-wysiwyg.git"],
```

Another slight format error. 

```suggestion
    "repository": "https://github.com/jpuri/react-draft-wysiwyg.git",
```

Hopefully the last :crossed_fingers: